### ISSUE TITLE
report_participant: a report-like output with tables etc.

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/report#61]
Currently returns only text. should we change its ouput to a "report"-like output with tables, long-text etc.?

**Comment by strengejacke** [comment]
Yes, why not? Would be some overlap to `report_sample()`, but that's not a problem I would say.
